src/helix-ui/styles/elements/hx-tabcontent.less

```diff
@@ -1,8 +1,11 @@
 hx-tabcontent {
-  background-color: inherit;
+  background-color: @gray-25;
```

Comment:
@palindromebart @racksmangonzalez: Design Review Requested

We changed to default `tab-content` background-color to **gray-25** so that the content is highlighted when selected. Any thoughts?

<img width="1421" alt="pr324-2-tabcontent" src="https://user-images.githubusercontent.com/10120600/45115593-e89f8c80-b115-11e8-9302-69bcfccf5730.png">

Reply:
Reverting changes